posthog/temporal/data_imports/workflow_activities/import_data_sync.py

```diff
@@ -349,15 +349,17 @@ def import_data_activity_sync(inputs: ImportDataActivityInputs):
             source = mongo_source(
                 connection_string=mongo_config.connection_string,
                 collection_names=endpoints,
-                is_incremental=schema.is_incremental,
                 logger=logger,
-                team_id=inputs.team_id,
-                incremental_field=schema.sync_type_config.get("incremental_field") if schema.is_incremental else None,
+                is_incremental=schema.should_use_incremental_field,
```

Comment:
Nit, can you change `is_incremental` to be `should_use_incremental_field` please - I changed all the sources on Fri to be like this